controllers/books_controller.js

```diff
@@ -0,0 +1,104 @@
+const { query } = require('express');
+const db = require('../models');
+
+const Book = db.books;
+
+const controller = {};
+
+controller.index = async (req, res, next) => {
+  try {
+    const books = await Book.findAll();
+    res.render('index', { title: 'Books', books });
+  } catch (err) {
+    next(err);
+  }
+};
+
+controller.new = (req, res, next) => {
+  res.render('bookedit', {
+    title: 'Add book',
+    docreate: true,
+    notekey: '',
+    book: undefined,
+  });
+};
+
+controller.show = async (req, res, next) => {
+  try {
+    const book = await Book.findOne({ where: { id: req.query.id } });
+    if (!book) {
+      throw new Error(`No book found for ${req.query.id}`);
+    } else {
+      res.render('bookshow', {
+        title: book ? book.title : '',
+        id: req.query.id,
+        book,
+      });
+    }
+  } catch (err) {
+    next(err);
+  }
+};
+
+controller.create = async (req, res, next) => {
+  try {
+    if (req.body.docreate === 'create') {
+      await Book.create({
+        title: req.body.title,
+        author: req.body.author,
+        publication_date: req.body.publication_date,
+        abstract: req.body.abstract,
+        book_cover: req.body.book_cover,
+      });
```

Comment:
I think you could destructure de body param and you could clean the method a little bit, 
something like: 

```suggestion
controller.create = async (req, res, next) => {
  const { body: { title, author, publication_date, abstract, book_cover, id, docreate } } = req;
  try {
    if (docreate === 'create') {
      await Book.create({
        title: title,
        author: author,
        publication_date: publication_date,
        abstract: abstract,
        book_cover: book_cover,
      });
```
the same could apply for some other methods in the file